scripts/install-cni.sh

```diff
@@ -30,6 +30,20 @@ cilium_ready() {
   # Send errors to stdout since they're "expected" errors.
   # This redirection doesn't affect exit status after execution.
   CNI_COMMAND=VERSION /host/home/kubernetes/bin/cilium-cni 2>&1
+
+  # Wait upto 60s for the cilium pod to report healthy.
+  echo "Waiting for upto 60s for cilium to report readiness"
```

Comment:
Move this after the inotify call instead of inside it. We don't want to start checking this before cilium-cni binary is written, I think.

Reply:
Done